README.md

```diff
@@ -19,6 +19,14 @@ Commands:
 python3 -m venv venv
 source ./venv/bin/activate
 pip install -r requirements.txt
+./debug-app.py
+```
+
+Go to http://localhost:5000/en/ in the browser.
+
+## Build
+
+```bash
 python3 generate_sites.py
 python3 -m http.server
```

Comment:
with those changes it doesn't work locally, becaues mapjs file is used without the build path in it on the site - to make it work this change should be introduced (plus the last line in the README should be changed - the /build should be cut from the path then)

```suggestion
python3 -m http.server --directory build
```